apps/drain-safe/src/App.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { Button, Loader, Title, TextField, Table, EthHashInfo } from '@gnosis.pm/safe-react-components';
+import { useSafeAppsSDK } from '@gnosis.pm/safe-apps-react-sdk';
+import Web3 from 'web3';
+import {AbiItem} from 'web3-utils';
+import erc20 from './abis/erc20';
+import { fetchJson } from './utils';
+import useServices from './hooks/useServices';
+import Container from './Container';
+import Icon from './Icon';
+import Flex from './Flex';
+
+interface IAsset {
```

Comment:
nitpick: we don't really use I* type of naming across the projects, would be good to stay consistent